platform/ios/src/MGLMapView.mm

```diff
@@ -1131,7 +1131,7 @@ - (void)setContentInset:(UIEdgeInsets)contentInset animated:(BOOL)animated compl
     else
     {
         _contentInset = contentInset;
-        [self didUpdateLocationWithUserTrackingAnimated:animated completionHandler:completion];
+        [self didUpdateLocationWithUserTrackingHeading:nil animated:animated completionHandler:completion];
```

Comment:
let's keep having `didUpdateLocationWithUserTrackingAnimated: completionHandler:` a convenience method which internally calls newly added `didUpdateLocationWithUserTrackingHeading:nil animated: completionHandler:` so that we don't change every place where heading is not available.

Reply:
These methods are all called by `didUpdateLocations` method to which I added another version. If we keep them as is, we should have our implementations of these methods.